datanode/networkhistory/snapshot/service_load_snapshot.go

```diff
@@ -79,6 +79,12 @@ func (b *Service) LoadSnapshotData(ctx context.Context, log LoadLog, currentStat
 	var totalRowsCopied int64
 	var rowsCopied int64
 	var removedConstraintsAndIndexes *constraintsAndIndexes
+
+	historyTableLastTimestampMap, err := getLastHistoryTimestampMap(ctx, b.connPool, dbMetaData)
```

Comment:
Why do we pass the `b.connPool` and not have `getLastHistoryTimestampMap` to be a method of `*Service` and simply use it directly?